src/Controller/DashboardController.java

```diff
@@ -149,17 +198,37 @@ else if (editUserType.getValue() == null && editUserNewPassword.getText().length
 		else if (editUserType.getValue() == "Guest") { type = UserType.GUEST; }
 		else if (editUserType.getValue() == "Stranger") { type = UserType.STRANGER; }
 
-		if (type == null) {
+		// Change user type if that had input
+		if (type != null) {
+			toChange.setType(type);
+			printToConsole("Successfully changed user type.");
+		}
+		// Change password if that had input
+		if (editUserNewPassword.getText().length() != 0) {
 			toChange.setPassword(editUserNewPassword.getText());
-			printToConsole("Set new password for chosen user.");
-			return;
+			printToConsole("Successfully changed user password.");
+		}
+
+		// Change location if that had input
+		if (editUserLocation.getValue().equals("Outside")) {
+			sim.getLoggedInUser().setLocation(null);
+		}
+		else if (editUserLocation.getValue() != null) {
+			for (Room r : sim.getHouse().getRooms()) {
+				if (r.getName().equals(editUserLocation.getValue())) {
+					sim.getLoggedInUser().setLocation(r);
```

Comment:
I feel like there should be a check to see if the new location is the same as the old location.  But maybe this check should be down within the setLocation method within the User class.  Maybe we should make the printToConsole method public so that errors can be printed from inside Model classes such as User and Simulation.

Reply:
Hm I think you're right for moving the logic to the Simulation class. However, we definitely shouldn't move printToConsole. Model should include all the logic to handle the simulation, controller should be using the functions in model to handle the situations it's provided, but printing to the console doesn't have anything to do with the house system/model, it's only a method to give back alerts and messages on task completions and stuff.

TL:DR; let's resolve the PRs for now and I will see if it's possible to move the logic of user creation/editing/deletion into the simulation class as it fits better in the MVC model.